### ISSUE TITLE
VirusTotal matches for windows executable

### ISSUE THREAD

**Issue by pfeerick** [g4ixt/QtTinySA#71]
Just downloaded the executable for Windows from https://github.com/g4ixt/QtTinySA/releases/tag/v0.11.8, but it appears to get a significant number of matches when checked with VirusTotal. I'm not saying there is a virus, but there are a significant number of matches, and both web Browser (Brave) and Windows 11 automatically blocked/deleted the file until instructed to leave it alone. It might be worth looking at using https://github.com/Nuitka/Nuitka-Action for transparency / reproducible builds. 

https://www.virustotal.com/gui/file/53b6fa7b36a13d5998a4a49cc879c169cc7380246d662452c7496c01cf50f638

**Comment by g4ixt** [comment]
Agreed, I intend to use a Nuitka action to build the executables.  The first step was in getting a successful 'local' compilation with Nuitka and I have done that for both Linux and Windows.  I will attempt it for the next version.
